manage.py

```diff
@@ -1,12 +1,23 @@
 #!/usr/bin/env python
+import os
 
-from coaster.manage import init_manager
+from coaster.manage import manager, init_manager
 
 from imgee.models import db
 from imgee import app
 
 
+def mkdir_p(dirname):
+    if not os.path.exists(dirname):
+        os.makedirs(dirname)
+
+
+@manager.command
+def init():
+    mkdir_p(os.path.join(app.static_folder, app.config['UPLOADED_FILES_DIR']))
```

Comment:
This `os.path.join` call is in two places now. Best if it was in a function that's imported in both places.